src/main/java/org/fogbowcloud/saps/engine/core/archiver/Archiver.java

```diff
@@ -181,51 +99,49 @@ private void garbageCollector() {
 
         for (SapsImage task : failedTasks)
             deleteAllTaskFilesFromDisk(task);
-
     }
 
     /**
-     * This function gets tasks in failed state in Catalog.
+     * It gets tasks in failed state in {@code Catalog}.
      *
-     * @return tasks in specific state
+     * @return {@code SapsImage} list in {@code ImageTaskState.FAILED} state
      */
     private List<SapsImage> tasksInFailedState() {
         return CatalogUtils.getTasks(catalog, ImageTaskState.FAILED,
                 "gets tasks with " + ImageTaskState.FAILED.getValue() + " state");
     }
 
     /**
-     * This function cleans unfinished data from incomplete archiving.
+     * It cleans unfinished data from incomplete {@code ImageTaskState.ARCHIVING}.
```

Comment:
you should explain in the docs what "clean" means in this context

Reply:
Done.